ModularTegustation/tegu_items/representative/items/ncorp.dm

```diff
@@ -63,3 +63,308 @@
 	amount = 80
 	allowed_roles = list("Records Officer", "Extraction Officer")
 
+//Temporary attributes
```

Comment:
Look at A Quiet Day actually on how to cut lines by like half

Reply:
Took the code right out of Flesh Idols realised ability which may have been a bad idea, also fixes a bug with the K-Corp research text while at it and formatted N-Corp text a bit better.